backend/testUtils/testSession.ts

```diff
@@ -21,6 +21,27 @@ export const mockTestSession: TestSessionRequestDTO = {
   startTime: new Date("2021-09-01T09:00:00.000Z"),
 };
 
+export const mockTestSessionsWithSameTestId: Array<TestSessionRequestDTO> = [
```

Comment:
we could merge this with `mockTestSession` and just add a second object. but two vars might also be more clear. i guess as long as we are consistent, we can go with either one

Reply:
yeah that was my reasoning - to keep the data for the test cases separate